### PR TITLE
docs: Add description to undocumented methods of the Integer class.

### PR DESCRIPTION
**Pull request by mttvn**
## Purpose and Motivation

Add description to undocumented methods of the Integer class.

## Types of changes

- Documentation

## To-do list

- [x] Code is tested
- [x] All tests are passing
- [x] Updated documentation
- [x] This PR is ready for review

### CODE REVIEW

**Review by joshpar — APPROVED** [review]

**Comment by joshpar** [comment]
closing to reopen to rerun CI tests

**Comment by joshpar** [comment]
Reopening

**Comment by dyfer** [comment]
@joshpar @jrsurge are we waiting for further review or is this good to be merged?

File: HelpSource/Classes/Integer.schelp [hunk 1]
```diff
@@ -204,6 +217,30 @@ code::
...
+LIST::
+## the receiver if it is between code::lo:: and code::hi::
+## code::lo:: if the receiver is less than code::lo::
+## code::hi:: if the receiver is greater than code::hi::
+::
+The result is an Integer only if both code::lo:: and code::hi:: are Integers.
+discussion::
+code::
+4.clip(2,5); // returns 4
+0.clip(2,5); // returns 0
```
Comment (telephon) [inline HelpSource/Classes/Integer.schelp:240]: should be 
`0.clip(2,5); // returns 2`

File: HelpSource/Classes/Integer.schelp [hunk 2]
```diff
@@ -213,9 +250,75 @@ code::
...
+Wraps in to a value between code::lo:: and code::hi::.
+returns:: an Integer between code::lo:: and code::hi::.
+discussion::
+code::
+(0..12).wrap(0,3); // returns [ 0, 1, 2, 3, 0, 1, 2, 3, 0, 1, 2, 3, 0 ]
+::
 
 subsection:: Misc
 
+method::archiveAsCompileString
```
Comment (telephon) [inline HelpSource/Classes/Integer.schelp:280]: This method is part of an interface that is not documented for any class. Maybe lik to code::Classes/Archive:: ?

(but I don't mind).

**Review by telephon — CHANGES_REQUESTED** [review]

**Comment by dyfer** [comment]
I'm going to push a commit addressing the review

**Comment by dyfer** [comment]
@telephon please let me know if this is good now. I decided to hide the `archiveAsCompileString` method in the help.

**Pull request merged**